src/main/java/com/nike/cerberus/domain/input/SecurityGroups.java

```diff
@@ -1,33 +0,0 @@
-/*
```

Comment:
Seems like this tagged along, this has nothing to do with this PR, it is a not used file though.